src/store.ts

```diff
@@ -371,18 +394,25 @@ export const initializeFirestoreListeners = (onLoad: () => void): (() => void) =
     .get()
     .then(snapshot => {
       const data = snapshot.data();
-      if (data != null) {
-        store.commit('setSemesters', data.semesters);
+      if (data) {
+        const { orderByNewest, semesters } = data;
+        store.commit('setSemesters', semesters);
+        // if user hasn't yet chosen an ordering, choose true by default
+        store.commit('setOrderByNewest', orderByNewest === undefined ? true : orderByNewest);
```

Comment:
```suggestion
        if (orderByNewest === undefined) {
            store.commit('setOrderByNewest', true);
        }
```
to avoid unnecessary `store.commit` calls.

Reply:
I committed this change but then found the user's choice was no longer persistent--any thoughts?